en/docs/tutorials/edit-an-api-by-modifyng-the-api-definition.md

```
@@ -153,17 +153,17 @@ In this tutorial, let's see how you can add interactive documentation to an API
     
     3. Click **Update Content**.
        This adds a resource with two HTTP methods into the API, which is visible in the API Publisher 
-       **Resources** tab along with the parameters defined. 
+       **Resources** tab, along with the parameters defined. 
        ![]({{base_path}}/assets/img/learn/tutorials/create-rest-api-pizzashack-resources.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)